0004/README.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 0004
+title: Tinkerbell Security
```

Comment:
I would expect a "Security" proposal to focus on the practices around security disclosures (reception and transmission) and the handling of the sensitive patches.

I think a better title for this would be "Network Interfaces and Visibility of Tinkerbell Services".